py_zipkin/thrift/__init__.py

```diff
@@ -146,6 +153,8 @@ def create_span(
         'binary_annotations': binary_annotations,
         'timestamp': int(timestamp_s * 1000000) if timestamp_s else None,
         'duration': int(duration_s * 1000000) if duration_s else None,
+        'trace_id_high': unsigned_hex_to_signed_int(trace_id_high)
+        if trace_id_high else None,
```

Comment:
oh I see now you actually did in the initial version.  👎  flake8. It's still kind of hard to read, so maybe just create this in a temporary variable before the span_dict line?